free-courses-es.md

```diff
@@ -0,0 +1,246 @@
+* [Organización y gestión del proyecto (2009)](https://ocw.unican.es/course/view.php?id=207) - Univ. de Cantabria
+* [Procesadores de Lenguaje (2012)](https://ocw.unican.es/course/view.php?id=158) - Univ. de Cantabria
+* [Sistemas operativos avanzados - 'scrum - bsd- Qt' (2015)](https://campusvirtual.ull.es/ocw/course/view.php?id=119) - Univ. de La Laguna
+
+
+### Ofimática
+
+* [Creación y retoque de imágenes con software libre](https://miriadax.net/web/creacion-y-retoque-2-ed/) - Univ. de Málaga
+* [Excel 1 - Básico](https://www.edx.org/course/excel-upvalenciax-xls101x-1) - Univ. de Valencia
+* [Excel 2 - Gestión de Datos](https://www.edx.org/course/excel-2-gestion-de-datos-upvalenciax-xls201x) - Univ. de Valencia
+* [OpenOffice](https://miriadax.net/web/software-libre-ofimatica-con-openoffice) - Univ. Politécnica de Valencia
+* [OpenOffice Calc. Gestión de datos sobre hojas de cálculo](https://ocw.unican.es/course/view.php?id=61) - Univ. de Cantabria (2014)
+* [Presentaciones eficaces (2012)](https://ocw.unican.es/course/view.php?id=188) - Univ. de Cantabria
+* [Presentaciones eficaces con PowerPoint](https://www.edx.org/es/course/disena-presentaciones-eficaces-con-upvalenciax-ppt101x-0) - Univ. Politécnica de Valencia
+* [Subtitulación de Vídeos](https://miriadax.net/web/subtitular-en-linea-2-edicion-) - Univ. Central de Cataluña
+
+
+### Procesadores de lenguaje
+
+* [Compiladores e Intérpretes (2012)](http://ocw.uji.es/curso/4949) - Univ. de Castellón (Jaime I)
+* [Procesadores de lenguaje (2006)](https://ocw.ua.es/es/ingenieria-y-arquitectura/procesadores-de-lenguaje-2006.html) - Univ. de Alicante
+* [Procesadores de Lenguaje (2012)](http://ocw.uji.es/curso/5180) - Univ. de Castellón (Jaime I)
+* [Procesadores de lenguajes -enfocado en Perl- ](https://campusvirtual.ull.es/ocw/course/view.php?id=45) - Univ. de La Laguna
+* [Procesadores de lenguajes II](https://ocw.uca.es/course/view.php?id=56) - Univ. de Cádiz
+
+
+### Programación
+
+* [Fundamentos de informática en lenguaje C - I](https://ocw.uca.es/course/view.php?id=31) - Univ. de Cádiz
+* [Fundamentos de informática en lenguaje C y Arduino - II](https://ocw.uca.es/course/view.php?id=74) - Univ. de Cádiz
+* [Introducción a la programación orientada a objetos en Java](https://www.coursera.org/learn/introduccion-programacion-java) - Univ. de los Andes
+* [Introducción a la programación para ciencias e ingieniería](https://miriadax.net/web/introduccion-programacion-ciencias-ingenieria-2edicion) - Univ. Politécnica de Madrid
+* [Introducción a la programación, Python I](https://www.coursera.org/learn/aprendiendo-programar-python) - Univ. Católica de Chile
+* [Introducción a la programación. Descubre el lenguaje de la era digital](https://miriadax.net/web/introduccion-a-la-programacion-descubre-el-lenguaje-de-la-era-digital-5-edicion-/) - Telefónica Universitas
+* [Introducción a Perl] (2012(https://campusvirtual.ull.es/ocw/course/view.php?id=43) - Univ. de La Laguna
```

Comment:
should be 
* [Introducción a Perl (2012)](https://campusvirtual.ull.es/ocw/course/view.php?id=43)